src/samples/ExtendedSlashScreen.Uno.Samples/ExtendedSlashScreen.Uno.Samples.Windows/ExtendedSlashScreen.Uno.Samples.Windows.csproj

```diff
@@ -0,0 +1,70 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
+		<TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>
+		<TargetPlatformMinVersion>10.0.18362.0</TargetPlatformMinVersion>
+		<RootNamespace>ExtendedSlashScreen.Uno.Samples</RootNamespace>
+		<ApplicationManifest>app.manifest</ApplicationManifest>
+		<Platforms>x86;x64;arm64</Platforms>
+		<RuntimeIdentifiers>win10-x86;win10-x64;win10-arm64</RuntimeIdentifiers>
+		<PublishProfile>win10-$(Platform).pubxml</PublishProfile>
+		<UseWinUI>true</UseWinUI>
+		<EnablePreviewMsixTooling>true</EnablePreviewMsixTooling>
+		<DefineConstants>$(DefineConstants);WINUI</DefineConstants>
```

Comment:
is this still useful now we do have `HAS_UNO_WINUI`

Reply:
When I replaced WINUI by HAS_UNO_WINUI it didnt work.